crystals/atom.py

```diff
@@ -159,6 +159,7 @@ def __init__(
         coords: ArrayLike,
         lattice: Optional[Union[Lattice, ArrayLike]] = None,
         displacement: ArrayLike = None,
+        meanSquareDisplacement: float = 0.0,
```

Comment:
Code throughout this library uses the `camel_case` convention, i.e. `mean_square_displacement` instead of `meanSquareDisplacement`. Please follow the existing convention